src/assets/server/rest/v1/schemas/chargingstation/chargingprofile-create.json

```diff
@@ -1,5 +1,115 @@
 {
   "title": "Create a Charging Profile",
   "type": "object",
-  "$ref": "chargingstation.json#/definitions/chargingProfile"
+  "properties": {
+    "chargingStationID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "connectorID": {
+      "$ref": "chargingstation.json#/definitions/connector/properties/connectorId"
+    },
+    "chargePointID": {
+      "type": "number",
```

Comment:
The connector ID is reused. Is there a reason, that the cchargePoint ID is not reused?